build.sh

```
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+USERNAME="madhusudhan143"
+DEV_TAG="$USERNAME/mern-thinkboard-dev"
+PROD_TAG="$USERNAME/mern-thinkboard-prod"
```

Shellcheck (0.10.0)

[warning] PROD_TAG appears unused. Verify use (or export if used externally).

(SC2034)